Source/WebCore/platform/graphics/ShareableBitmap.h

```diff
@@ -190,7 +190,7 @@ class ShareableBitmap : public ThreadSafeRefCounted<ShareableBitmap> {
     WEBCORE_EXPORT RefPtr<cairo_surface_t> createCairoSurface();
 
     PlatformImagePtr createPlatformImage(BackingStoreCopy = CopyBackingStore, ShouldInterpolate = ShouldInterpolate::No) { return createCairoSurface(); }
-#elif USE(HAIKU)
+#elif PLATFORM(HAIKU)
```

Comment:
why is this a PLATFORM(HAIKU)?

This would be a problem if trying to build gtkwebkit on Haiku for example (you want to end up in USE(SKIA) in that case))

Reply:
Yes, but there's also USE(HAIKU), and here the other cases are all defined with USE() so we could probably do the same.

Later on, maybe USE() can be split into more specific parts like USE(HAIKU_NETWORK), USE(HAIKU_GRAPHICS), etc, to make it easier to enable or disable just some parts.